source/features/batch-open-conversations.tsx

```diff
@@ -63,12 +63,49 @@ async function init(): Promise<void | false> {
 				Open selected
 			</button>
 		);
+	} else {
+		const issuesToolbar = select('#js-issues-toolbar')!;
+		issuesToolbar.prepend(
+			<div className="mr-3 d-none d-md-block">
+				<input type="checkbox" data-check-all="" aria-label="Select all issues" autoComplete="off"/>
+			</div>
+		);
+		issuesToolbar.append(
+			<div className="table-list-triage flex-auto js-issues-toolbar-triage">
+				<span className="text-gray table-list-header-toggle">
+					<span data-check-all-count="">1</span> selected
+					{' '}
+					<button type="button" className="btn-link rgh-batch-open-issues pl-3">Open selected</button>
+				</span>
+			</div>
+		);
+
+		const regExpPrNumber = /^issue_(\d*)$/;
+
+		const prRows = select.all('.js-issue-row');
+		for (const prRow of prRows) {
+			const prNumber = regExpPrNumber.exec(prRow.id)?.[1] ?? '';
+			const prependAt = prRow.children[0]!;
+			prependAt.prepend(
+				<label className="flex-shrink-0 py-2 pl-3  d-none d-md-block">
+					<input
+						type="checkbox"
+						data-check-all-item=""
```

Comment:
Does this work?
```suggestion
						data-check-all-item
```

Reply:
with this change i get from normal linter in WebStorm: `Wrong attribute value` 

> HtmlUnknownBooleanAttribute
> Unknown HTML boolean tag attribute
> This inspection highlights HTML non-boolean tag attributes without value as invalid, and lets mark such attributes as Custom to avoid highlighting them as invalid.

the XO linter not complain

Reply:
Yeah, WebStorm is stricter, but dom-chef should support it, which is what matters.

Reply:
and similar is already in codebase:

https://github.com/sindresorhus/refined-github/blob/4248fcf8a430093077c5109e6e5f675c6a068045/source/features/show-associated-branch-prs-on-fork.tsx#L80-L82

i change it